stl/inc/format

```diff
@@ -2355,14 +2348,12 @@ public:
 
 inline constexpr size_t _Fmt_buffer_size = 256;
 
-template <class _Iterator>
-struct _Back_insert_iterator_container_type {
-    using type = void;
-};
-
-template <class _Container>
-struct _Back_insert_iterator_container_type<back_insert_iterator<_Container>> {
-    using type = _Container;
+template <class _OutputIt>
+struct _Fmt_iterator_flush {
+    template <class _Ty>
+    static _OutputIt _Flush(const _Ty* _First, const _Ty* _Last, _OutputIt _Output) {

```

Comment:
'_Output` potentially moves, but the pointers can be top-level const

Reply:
Added